definitions/json/EventToComplexTypes/EventToComplexTypes.json

```diff
@@ -2749,7 +2749,7 @@
     "CaseEventID": "amendCaseDetails",
     "CaseFieldID": "claimantHearingPreference",
     "ListElementCode": "hearing_preferences",
-    "EventElementLabel": "What are the claimant's hearing preferences",
+    "EventElementLabel": "What are the claimant's hearing preferences2",
```

Comment:
I assume the number at the end was for test purposes and can be removed?

Reply:
```suggestion
    "EventElementLabel": "What are the claimant's hearing preferences",
```